packages/attention/index.js

```diff
@@ -1,76 +1,163 @@
 import { css, html } from 'lit';
-import { FabricElement, fclasses } from '../utils';
+import { FabricElement, classes } from '../utils';
+import { attention as c } from '@fabric-ds/css/component-classes';
+import { opposites, rotation, useRecompute as recompute } from '@fabric-ds/core/attention';
 
 class FabricAttention extends FabricElement {
+  static properties = {
+    // Render Attention element without arrow
+    noArrow: { type: Boolean, reflect: true },
```

Comment:
It might be more idiomatic to use kebab case for these props? Or not... In any case, that can be done by setting attribute: 'some-name'

Reply:
Good point! Changed in [8831818](https://github.com/fabric-ds/elements/pull/71/commits/8831818e7eea51e618ab97cf75560dcc3c2dd54d) based on [this Medium article](https://medium.com/@dayton-bobbitt/generating-attributes-for-litelement-properties-f972ef658137)